tensorflow/c/c_api_experimental.h

```diff
@@ -304,6 +304,25 @@ TF_CAPI_EXPORT extern void
 TF_ImportGraphDefOptionsSetValidateColocationConstraints(
     TF_ImportGraphDefOptions* opts, unsigned char enable);
 
+// Load the library specified by library_filename and register the pluggable
```

Comment:
Please add a comment as to which platforms support this. Looking at the code, for example, Mobile and Slim for example, are unsupported. This should be reflected in the doc comments here.

Reply:
done